pkg/deploy/tls.go

```diff
@@ -330,14 +327,14 @@ func K8sHandleCheTLSSecrets(deployContext *DeployContext) (reconcile.Result, err
 			return reconcile.Result{RequeueAfter: time.Second}, err
 		}
 
-		role, err := SyncTLSRoleToCluster(deployContext)
-		if role == nil {
-			return reconcile.Result{RequeueAfter: time.Second}, err
+		done, err := SyncTLSRoleToCluster(deployContext)
+		if !done {
+			return reconcile.Result{}, err
 		}
 
-		roleBiding, err := SyncRoleBindingToCluster(deployContext, CheTLSJobRoleBindingName, CheTLSJobServiceAccountName, CheTLSJobRoleName, "Role")
-		if roleBiding == nil {
-			return reconcile.Result{RequeueAfter: time.Second}, err
+		done, err = SyncRoleBindingToCluster(deployContext, CheTLSJobRoleBindingName, CheTLSJobServiceAccountName, CheTLSJobRoleName, "Role")
+		if !done {
+			return reconcile.Result{}, err
```

Comment:
Maybe log error:
``` go
			if err != nil {
				logrus.Error(err)
			}
			return reconcile.Result{RequeueAfter: time.Second}, err
```

Reply:
Error will be printed by che_controller.go
https://github.com/eclipse-che/che-operator/blob/master/pkg/controller/che/che_controller.go#L511-L519